library/VTKExtensions/Rendering/vtkF3DRenderer.h

```diff
@@ -267,8 +278,13 @@ class vtkF3DRenderer : public vtkOpenGLRenderer
 
   bool HasValidHDRIFile = false;
   std::string HDRIFile;
+  bool HasValidHDRIHash = false;
   std::string HDRIHash;
+  bool HasValidHDRITexture = false;
   vtkNew<vtkTexture> HDRITexture;
+  bool HasValidHDRILUT = false;
+  bool HasValidHDRISH = false;
+  bool HasValidHDRISpec = false;
```

Comment:
`HasValidHDRI*` is different from `HDRI*Configured` in the sense that valid means it is valid while configured means will be set to true whatever happens when Configure method is called.

It is needed to ensure we can turn on/off IBL or Skybox without recomputing anything while still being able to change the HDRI and actually recompute the IBL textures cache and such.